lgc/patch/PatchInOutImportExport.cpp

```diff
@@ -3465,15 +3471,15 @@ void PatchInOutImportExport::createStreamOutBufferStoreFunction(Value *storeValu
 
   format = formatOprd.u32All;
```

Comment:
Does it also make sense to refactor the preceding switch, where for gfx10 we needlessly compute formatOprd?

Reply:
Yes. I've rewritten more of this function.

Reply:
Thanks.